frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -18,7 +18,11 @@ data class SlideOutOptions(
 class DeleteBranchOnSlideOutSuggestionDialog(project: Project) :
     DialogWrapper(project, /* canBeParent */ true) {
 
-  private var remember = false
+  // Must NOT be private to prevent:
+  // java.lang.IllegalAccessException: class
```

Comment:
In line 84, no idea why so 😯 as it used to work on kotlin 1.3

Reply:
Whoops there are more places like that... SlideInDialog has the same runtime error... not good...

Reply:
Ok, I banned all private fields in Kotlin code xDDDDD
No idea what would be/if there is any better solution ;p